compiler/rustc_attr/src/builtin.rs

```diff
@@ -720,17 +721,37 @@ pub fn eval_condition(
 
 #[derive(Copy, Debug, Encodable, Decodable, Clone, HashStable_Generic)]
 pub struct Deprecation {
-    pub since: Option<Symbol>,
+    pub since: Option<DeprecatedSince>,
```

Comment:
Should we add a Missing variant and remove the Option?

Reply:
`Missing` would make some things more complicated because then it no longer makes sense for `DeprecatedSince` to implement `Display`, which rustdoc currently relies on.

Rustdoc may already need a different approach upon introducing an `Err` variant so I'll take a look.

Reply:
After introducing `Err`, eliminating the `Option` indeed seems like the right call. :+1: